third_party/Makefile

```diff
@@ -6,10 +6,13 @@
 SB_MICRO_PATH ?= "/usr/local"
 MPI_HOME_PATH ?= "/usr/local/mpi"
 
-.PHONY: all cutlass bandwidthTest nccl_tests perftest fio
+.PHONY: all cuda rocm common cutlass bandwidthTest nccl_tests perftest_cuda perftest_rocm fio
```

Comment:
For microbenchmark, I remember we have put cuda/rocm at the beginning?

Reply:
ok, I will unify all target names and put cuda/rocm as prefix